documentation/Command-summary.md

```diff
@@ -423,7 +425,52 @@ if __name__ == "__main__":
         print(data_object_hash, flush=True)
 ```
 
-Because datasets are unordered collections, an ordering or sorting operation is most useful when combined with a following filter operation such as `--limit`.
+Scripts should be created and called the same way they would be called on the commandline. For example a python script like the one above could be run the following ways:
+
+#### Platform independent
+
+Place python code in `reverse.py` and call with `python3 path/to/reverse.py`. The disadvantage of this method is that you have to specify the full path to `reverse.py`.

```

Comment:
It's the full path requirement true? It works for me with relative paths

Reply:
Yeah, I just meant that you have to specify the path instead of just the name of the script. I will remove the word "full" as it's misleading